package.json

```diff
@@ -118,7 +118,7 @@
     "@types/node": "16.11.39",
     "@types/node-fetch": "2.5.12",
     "@types/node-forge": "1.3.1",
-    "@types/plist": "3.0.2",
+    "@types/plist": "^3.0.2",
```

Comment:
I believe we went with pinning exact versions of dependencies (rather than ranges) due to reasons relating to `@rancher/components`. If this is intentional, please put it in a separate commit and pass it by Phil.

Reply:
I just rebased it, and it looks like it got fixed.